File: app/src/main/java/se/kicksort/metalarchives/model/AllBandMembers.java
package se.kicksort.metalarchives.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

/**
 * Created by Gustav Sundin on 2017-04-28.
 */

public class AllBandMembers {
    @SerializedName("current")
    @Expose
    private ArrayList<BandMember> currentLineup;

    @SerializedName("past")
    @Expose
    private ArrayList<BandMember> pastLineup;

    @SerializedName("live")
    @Expose
    private ArrayList<BandMember> liveLineup;

    public ArrayList<BandMember> getCurrentLineup() {
        return new ArrayList<>(currentLineup);
    }
}

File: app/src/main/java/se/kicksort/metalarchives/model/AllBandMembers.kt
package se.kicksort.metalarchives.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.util.ArrayList

/**
 * Created by Gustav Sundin on 2017-04-28.
 */

class AllBandMembers {
    @SerializedName("current")
    @Expose
    val currentLineup: ArrayList<BandMember>? = null

    @SerializedName("past")
    @Expose
    val pastLineup: ArrayList<BandMember>? = null

    @SerializedName("live")
    @Expose
    val liveLineup: ArrayList<BandMember>? = null
}
